testing/integration/beat_receivers_test.go

```diff
@@ -480,3 +481,159 @@ service:
 	})
 
 }
+
+// TestAgentMetricsInput is a test to provide a baseline for what
+// elastic-agent metrics looks like with default input metrics.  It
+// will be expanded in the future to compare with beats receivers.
+func TestAgentMetricsInput(t *testing.T) {
+	info := define.Require(t, define.Requirements{
+		Group: Default,
+		Local: true,
+		OS: []define.OS{
+			{Type: define.Windows},
+			{Type: define.Linux},
+			{Type: define.Darwin},
+		},
+		Stack: &define.Stack{},
+	})
+	tmpDir := t.TempDir()
+
+	timestamp := time.Now().UTC().Format("2006-01-02T15:04:05.000Z")
+
+	type configOptions struct {
+		HomeDir         string
+		ESEndpoint      string
+		ESApiKey        string
+		BeatsESApiKey   string
+		FBReceiverIndex string
+		Namespace       string
+	}
+	esEndpoint, err := getESHost()
+	require.NoError(t, err, "error getting elasticsearch endpoint")
+	esApiKey, err := createESApiKey(info.ESClient)
+	require.NoError(t, err, "error creating API key")
+	require.True(t, len(esApiKey.Encoded) > 1, "api key is invalid %q", esApiKey)
```

Comment:
It would be a good practice not to log an esApiKey (even if it's just a test one).
Furthermore, if you wanna format your assertion failure message, you probably need `require.Truef`

Reply:
I was under the assumption that CI test logs were private. Almost every test in this file and in otel_test.go print the message in the same way, we would have to fix those as well.